src/vm/codeman.cpp

```diff
@@ -3020,7 +3020,7 @@ void * EEJitManager::allocCodeFragmentBlock(size_t blockSize, unsigned alignment
     {
         CrstHolder ch(&m_CodeHeapCritSec);
 
-        mem = (TADDR) allocCodeRaw(&requestInfo, sizeof(TADDR), blockSize, alignment, &pCodeHeap);
+        mem = (TADDR) allocCodeRaw(&requestInfo, sizeof(CodeHeader), blockSize, alignment, &pCodeHeap);
```

Comment:
The issue that I mentioned above happens when ``WIN64EXCEPTIONS`` is set, but ``USE_INDIRECT_CODEHEADER`` is not set. Later I decide to set ``USE_INDIRECT_CODEHEADER`` for x86/Linux.

Is there any difference bettwen Windows and Linux? IMHO, this issue should be reproduced for x86/Windows port (which uses ditect code header), but it isn't.